Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean

```diff
@@ -397,6 +397,56 @@ lemma exists_isCycle_snd_verts_eq {p : G.Walk v v} (h : p.IsCycle) (hadj : p.toS
 
 end IsCycle
 
+/-- This lemma considers the `SimpleGraph.Walk` until any vertex in a given set. You could
```

Comment:
Can you give some more detail on what the lemma is actually saying here: at the moment the only actual descriptor is "considers", which doesn't help the reader much!

Reply:
Fixed to be more specific:)